tests/tests_lite/run_standalone_tests.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+# THIS FILE ASSUMES IT IS RUN INSIDE THE tests/tests_pytorch DIRECTORY
+
+# Batch size for testing: Determines how many standalone test invocations run in parallel
+# It can be set through the env variable PL_STANDALONE_TESTS_BATCH_SIZE and defaults to 6 if not set
+test_batch_size="${PL_STANDALONE_TESTS_BATCH_SIZE:-6}"
+
+# this environment variable allows special tests to run
+export PL_RUN_STANDALONE_TESTS=1
+# python arguments
+defaults='-m coverage run --source lightning_lite --append -m pytest --no-header'
```

Comment:
Maybe this could be a symlink where we make the source dir customizable

Reply:
Like the batch size through env variable?

Reply:
That would work

Reply:
Did that. Fingers crossed. [8f37c9d](https://github.com/Lightning-AI/lightning/pull/14451/commits/8f37c9daf28284aa5a68c2743966fa6837e68a72)